spec/support/saml_idp_bad_destination_controller.rb.erb

```diff
@@ -0,0 +1,187 @@
+class SamlIdpController < SamlIdp::IdpController
```

Comment:
This controller is the same as the template's normal controller, except that `Destination="{recipient}"` in all the places that `Destination="#{@saml_slo_acs_url}"` which results in an invalid response and causes `Devise::Strategies::SamlAuthenticatable#authenticate!` to fail in the new sad path integration specs.